readthedocs/rtd_tests/tests/test_profile_views.py

```diff
@@ -35,6 +35,35 @@ def test_edit_profile(self):
         self.assertEqual(self.user.last_name, 'Docs')
         self.assertEqual(self.user.profile.homepage, 'readthedocs.org')
 
+    def test_edit_profile_with_invalid_values(self):
+        resp = self.client.get(
+            reverse('profiles_profile_edit'),
+        )
+        self.assertTrue(resp.status_code, 200)
+
+        long_first_name = 'this-is-first_name-whose-length-is-greater-than-30'
+        long_last_name = 'this-is-last_name-whose-length-is-greater-than-30'
+        long_homepage = 'this-is-the-the-link-to-the-homepage-of-the-user-which-is-pretty-long-and-that-is-why-wrong-value.org'
+
+        self.assertTrue(len(long_first_name) > 30)
+        self.assertTrue(len(long_last_name) > 30)
+        self.assertTrue(len(long_homepage) > 100)
```

Comment:
we could just use string multiplication to make the test shorter and more clear, `'a' * 31`

Reply:
Thanks for guiding me on this.
I have made the changes.